lusidtools/lpt/qry_properties.py

```diff
@@ -48,4 +48,5 @@ def success(result):
 
 # Standalone tool
 def main(parse=parse, display_df=lpt.display_df):
-    return lpt.standard_flow(parse, lse.connect, process_args, display_df)
+    raise NotImplementedError("Jira: SENG-40")
```

Comment:
If you want we could also comment out TOOLNAME and TOOLTIP. This will prevent the Lusidtools cli from identifying it as a function, removing it from things like the lusidtools --help. Its probably fine to leave as not implemented, but its just another option

Reply:
Can we also add a bit more description around this for anyone who doesnt have access to our Jira? something like 
Jira: SENG-40 - Property Search was updated and this is now deprecated and requires updating